CHANGES.txt

```diff
@@ -1,6 +1,12 @@
 CHANGES
 =======
 
+0.4.6 (2023-02-23)
```

Comment:
Why did you add this date here?

Reply:
I forgot to change it 😄  to today's date. Or should it not be included until release? That would make sense.